### PR TITLE
[RHACS] Release notes for 3.73.0

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://github.com/stackrox/stackrox/blob/master/CHANGELOG.md

- Preview at https://openshift-docs-git-373relnotes-gnelson.vercel.app/openshift-acs/master/release_notes/373-release-notes.html

- Merge into `rhacs-docs-3.73.0` doesn't require cherrypicks.

### CODE REVIEW

File: release_notes/373-release-notes.adoc [hunk 1]
```diff
@@ -15,22 +15,73 @@ toc::[]
...
+* xref:../configuration/configure-proxy.adoc[Configuring a proxy for external network access]
+* xref:../configuration/configure-endpoints.adoc[Configuring endpoints]
+* xref:../configuration/monitor-acs.adoc[Monitoring with Prometheus]
+* xref:../operating/manage-user-access/configure-ocp-oauth.adoc[Configuring OpenShift Container Platform OAuth server as an identity provider]
+* xref:../backup_and_restore/backing-up-acs.adoc[Backing up Red Hat Advanced Cluster Security for Kubernetes]
+* xref:../backup_and_restore/restore-acs.adoc[Restoring from a backup]
+====
+
+[id="vulnerability-dashboard-changes"]
+==== Vulnerability dashboard changes for Cloud Service users
```
Comment (Wiharris) [inline release_notes/373-release-notes.adoc:58]: I would reword this. Maybe: `Cloud Service Vulnerability Changes Dashboard`

File: release_notes/373-release-notes.adoc [hunk 2]
```diff
@@ -45,112 +96,157 @@ In the table, features are marked with the following statuses:
...
-| GA
-| DEP
+|====
 
+[id="removed-features-373"]
+=== Removed features
+
+This section provides additional information about some of the removed features listed in the previous table.
+
+* The `ROX_WHITELIST_GENERATION_DURATION` environment variable is removed in the RHACS 3.73 release. You can use `ROX_BASELINE_GENERATION_DURATION` instead.
```
Comment (Wiharris) [inline release_notes/373-release-notes.adoc:174]:
```suggestion
* The `ROX_WHITELIST_GENERATION_DURATION` environment variable is removed in {product-title-short} 3.73 release. You can use `ROX_BASELINE_GENERATION_DURATION` can now be used instead.
```

File: release_notes/373-release-notes.adoc [hunk 3]
```diff
@@ -45,112 +96,157 @@ In the table, features are marked with the following statuses:
...
+* Red Hat has removed the `ComplianceRunSchedule` resource, which RHACS did not use.
+* Permissions for permission sets are grouped for simplification. The following list describes the new permissions and indicates the removed permissions in the RHACS 3.73.0 release:
+
+** The `Access` permission replaces the following permissions: `AuthPlugin`, `AuthProvider`, `Group`, `Licenses`, `Role`, and `User`.
+** The `DeploymentExtension` permission replaces the following permissions: `Indicator`, `NetworkBaseline`, `ProcessWhitelist`, and `Risk`.
+* The `Integration` permission deprecates the following permissions: `APIToken`, `BackupPlugins`, `ImageIntegration`, `Notifier`, and `SignatureIntegration`.
+** The `Image` permission replaces the `ImageComponent` permission.
+* Central reaches out to Scanner on the `scanner.<namespace>.svc` endpoint instead of `scanner.<namespace>` to respect {ocp}’s `NO_PROXY` configuration. If you are using `NO_PROXY`, add `*.svc` or `scanner.<namespace>.svc` to your `NO_PROXY` configuration. (ROX-13034)
+* `Label` and `Annotation` search options are removed in the RHACS 3.73 release. They are replaced by the search options listed in the following table:
```
Comment (Wiharris) [inline release_notes/373-release-notes.adoc:187]: I would replace RHACS with {product-title-short} for all of these

File: release_notes/373-release-notes.adoc [hunk 4]
```diff
@@ -45,112 +96,157 @@ In the table, features are marked with the following statuses:
...
-== Bug fixes
 
-[id="resolved-in-version-3730"]
-=== Resolved in version 3.73.0
 
-Release date: 28 November 2022
+[id="upcoming-in-product-docs-removal"]
+=== Notice of upcoming in-product docs removal
+
+Beginning in the RHACS 3.74 release, Red Hat will remove the in-product docs accessible from the help menu.
```
Comment (Wiharris) [inline release_notes/373-release-notes.adoc:241]:
```suggestion
Beginning in the {product-title-short} 3.74 release, Red Hat will remove the in-product docs accessible from the help menu.
```

**Review by Wiharris — COMMENTED** [review]
/LGTM Left a few comments: 

Do you need a QE review for this PR as well? I'm assuming because it's release notes, it is not required but just making sure.

I would also fix the hardline wraps, particularly around lines 26-28 and lines 61-84 as well. :)

**Review by charmik-redhat — COMMENTED** [review]
As per 3.72 release notes' [upcoming API changes](https://docs.openshift.com/acs/3.72/release_notes/372-release-notes.html#upcoming-api-changes) section, CSV export API has changed for 3.73  and will now require `CVE Type` to be passed as part of the query parameter. Similarly, those using UI button for CSV export will need to add the `CVE Type` filter with one of IMAGE_CVE, K8S_CVE, ISTIO_CVE, NODE_CVE, and OPENSHIFT_CVE as value. The UI will show error if the filter is not provided.

Although we had mentioned this in 3.72 release notes' upcoming changes section, I think we should reiterate the same for 3.73 release notes too, perhaps in the notable technical changes section.

Cc @kcarmichael08

**Comment by gaurav-nelson** [comment]
@charmik-redhat Thank you, I'll include it in the next iteration.

File: release_notes/373-release-notes.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply release_notes/373-release-notes.adoc:58]: There is a  "Vulnerability dashboard" which now has some changes when people use Cloud Service. That suggestion would change its meaning, so I'm keeping it the same.

File: release_notes/373-release-notes.adoc [hunk 5]
```diff
@@ -15,22 +15,73 @@ toc::[]
...
-
-
 [id="new-features-373"]
 == New features
 
-[id="doc-updates-373"]
-=== Documentation updates
-
+[id="rhacs-cloud-service"]
+=== Red Hat Advanced Cluster Security Cloud Service
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply release_notes/373-release-notes.adoc:22]: The product name is "Red Hat Advanced Cluster Security Cloud Service"

**Pull request merged**